README.md

```diff
@@ -360,6 +360,9 @@ This class extends [`EventEmitter`][] from Node.js.
     `fs.close()`, and will close them automatically when the Worker exits.
     Defaults to `true`. (This option is only supported on Node.js 12.19+ and
     all Node.js versions higher than 14.6.0).
+  * `workerIdEnv`: (`boolean|string`) If `true`, sets an environment variable in
```

Comment:
I think that we should start simple, just using `workerId` should be enough to create the environment var with the id of the worker within.

Customizing the name might be interesting, but I'll rather prefer a separated API for it, maybe `workerIdName` or similar.